src/util.cpp

```diff
@@ -1502,7 +1502,7 @@ const wxString getProgramDescription(int type)
         if (display->IsPrimary())
         {
 
-            description << wxString::Format(bull + "%ix%i %ibit %ix%ippi\n",
+            description << wxString::Format(bull + "%ix%i %i-bit %ix%ippi\n",
```

Comment:
Can the newline please be added outside the translation?

From:
`description << wxString::Format(bull + "%ix%i %i-bit %ix%ippi\n",`
to:
`description << wxString::Format(bull + "%ix%i %i-bit %ix%ippi" + "\n",`

Thank you